convert2rhel/unit_tests/actions/pre_ponr_changes/backup_system_test.py

```diff
@@ -65,17 +96,120 @@ def test_backup_redhat_release_calls(self, backup_redhat_release_action, monkeyp
         assert backup_system.system_release_file.enable.call_count == 1
         assert backup_system.os_release_file.enable.call_count == 1
 
-    def test_backup_repository_calls(self, backup_repository_action, monkeypatch):
-        backup_varsdir_mock = mock.Mock()
-        backup_yum_repos_mock = mock.Mock()
+    @all_systems
+    def test_backup_repository_nonexisting_paths(
+        self, tmpdir, monkeypatch, caplog, backup_repository_action, pretend_os
+    ):
+        """Test empty paths, nothing for backup."""
+        etc = tmpdir.mkdir("etc")
+
+        monkeypatch.setattr(backup_system, "DEFAULT_YUM_REPOFILE_DIR", str(etc))
```

Comment:
We could simplify this to: 

```suggestion
        etc = tmpdir.mkdir("etc")

        monkeypatch.setattr(backup_system, "DEFAULT_YUM_REPOFILE_DIR", "/non-existing-path")
```

That would give the same effect, and much simpler as well.